src/components/JobWizard/JobWizardSteps/JobWizardFunctionSelection/JobWizardFunctionSelection.js

```diff
@@ -353,8 +359,24 @@ const JobWizardFunctionSelection = ({
     return openConfirmPopUp('All changes will be lost', confirmHandler)
   }
 
+  useEffect(() => {
+    if (!stepIsActive) {
+      setAutoSwitchedTab(true)
+      return
+    }
+    if (autoSwitchedTab) {
+      if (selectedActiveTab.current && selectedActiveTab.current !== activeTab) {
+        setActiveTab(selectedActiveTab.current)
+      }
+      setAutoSwitchedTab(false)
+    }
```

Comment:
I believe you don't need `autoSwitchedTab` state. You can change the activeStep using the condition `!stepIsActive && selectedActiveTab.current && selectedActiveTab.current !== activeTab`

Reply:
done.